src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -53,59 +118,30 @@ export const NFTListingsGrid = ({ selectedMarkets }: { selectedMarkets: ListingM
   )
 
   return (
-    <Column>
+    <GridWrapper>
       <TableHeader>
-        <Column
-          marginLeft="0"
-          transition="500"
-          className={bodySmall}
-          color="textSecondary"
-          flex={{ sm: '2', md: '1.5' }}
-        >
+        <NFTHeader>
           <Trans>NFT</Trans>
-        </Column>
-        <Row flex={{ sm: '1', md: '3' }}>
-          <Column
-            className={bodySmall}
-            color="textSecondary"
-            flex="1"
-            display={{ sm: 'none', xxl: 'flex' }}
-            textAlign="left"
-          >
+        </NFTHeader>
+        <PriceHeaders>
+          <PriceInfoHeader>
             <Trans>Floor</Trans>
-          </Column>
-          <Column
-            className={bodySmall}
-            color="textSecondary"
-            flex="1"
-            display={{ sm: 'none', xxl: 'flex' }}
-            textAlign="left"
-          >
+          </PriceInfoHeader>
+          <PriceInfoHeader>
             <Trans>Last</Trans>
-          </Column>
-          <Column className={subheadSmall} flex="2">
+          </PriceInfoHeader>
+
+          <DropdownWrapper>
             <SortDropdown dropDownOptions={priceDropdownOptions} mini miniPrompt={t`Set price by`} />
-          </Column>
-
-          <Column
-            className={bodySmall}
-            color="textSecondary"
-            flex="1"
-            display={{ sm: 'none', lg: 'flex' }}
-            textAlign="right"
-          >
+          </DropdownWrapper>
+
```

Comment:
are these whitespaces necessary?

Reply:
I wanted to separate the functional dropdown vs the static text headers to increase readability